build-system/tasks/process-github-issues.js

```diff
@@ -235,21 +307,59 @@ function applyMilestone(issue, milestoneNumber) {
  * @return {!Promise<*>}
  */
 function applyLabel(issue, label) {
-  const options = extend({}, milestoneOptions);
+  const options = extend({}, issuesOptions);
   options.qs = {
     'state': 'open',
     'per_page': 100,
     'access_token': GITHUB_ACCESS_TOKEN,
   };
-  return createGithubRequest(
-      '/issues/' + issue.number + '/labels',
-      'POST',
-      [label], 'label').then(function() {
-        if (verbose) {
-          util.log(util.colors.green(
-              'Label applied ' + label + ' for # ' + issue.number));
-        }
-      });
+  if (isDryrun) {
+    util.log(util.colors.green('Label applied ' +
+        label + ' for #' + issue.number));
+    return;
+  } else {
+    return createGithubRequest('/issues/' + issue.number + '/labels','POST',
+        [label], 'label');
+  }
+
+}
+
+/**
+ * @param {string} issue
+ * @param {string} comment
+ * @return {!Promise<*>}
+ */
+function applyComment(issue, comment) {
+  const options = extend({}, issuesOptions);
+  options.qs = {
+    'state': 'open',
+    'per_page': 100,
+    'access_token': GITHUB_ACCESS_TOKEN,
+  };
+  // delay the comment request so we don't reach github rate limits requests
```

Comment:
Does this mean that your gulp task will lie idle for 2 minutes? Perhaps print a message indicating the reason for the idle time?

Reply:
This wasn't addressed.